taverna-docker-activity/src/test/java/org/apache/taverna/activities/docker/test/TestDockerCommands.java

```diff
@@ -0,0 +1,101 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package org.apache.taverna.activities.docker.test;
+
+import com.github.dockerjava.api.command.InspectImageResponse;
+import com.github.dockerjava.api.model.Container;
+import org.apache.taverna.activities.docker.DockerContainerConfigurationImpl;
+import org.apache.taverna.activities.docker.DockerRemoteConfig;
+import org.apache.taverna.activities.docker.RemoteClient;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+public class TestDockerCommands {
+
+    private RemoteClient remoteClient;
+
+    private static final String IMAGE_NAME = "training/webapp";
+
+    private static final String CONTAINER_NAME = "test-container";
+
+    private static final String DOCKER_LOGIN_SUCCESS = "Login Succeeded";
+
+    public static void main(String[] args) {
+        TestDockerCommands commands = new TestDockerCommands();
+        commands.testLogin();
+//        commands.testInspectImage();
+        commands.testListContainers();
+//      commands.testCreateContainer();
+    }
+
+    public TestDockerCommands(){
+        DockerContainerConfigurationImpl containerConfiguration = new DockerContainerConfigurationImpl(new TestConfigurationManager());
+        containerConfiguration.getInternalPropertyMap().put(DockerContainerConfigurationImpl.NAME,CONTAINER_NAME);
+        containerConfiguration.getInternalPropertyMap().put(DockerContainerConfigurationImpl.IMAGE,IMAGE_NAME);
+        containerConfiguration.getInternalPropertyMap().put(DockerContainerConfigurationImpl.CMD,"env");
+
+        DockerRemoteConfig remoteConfig = new DockerRemoteConfig();
+        remoteConfig.setDockerHost("tcp://192.168.99.100:2376");
+        remoteConfig.setApiVersion("1.21");
+        remoteConfig.setDockerTlsVerify(true);
+        remoteConfig.setDockerCertPath("/Users/Nadeesh/Documents/docker/");
```

Comment:
Hi, @NadeeshDilanga  - could you try to avoid hardcoding `/Users/Nadeesh` here? Perhaps using the current directory instead?

I am not sure how I should test this.. what do I need to put in that folder? Could you add a README.md about what I need to copy where?